### PR TITLE
MBS-7669 Introduce a blog [MERGE]

### PR DESCRIPTION
**Pull request by eugene-krivobokov**
I've added a blog to post news and similar less structured information:

Screen Shot 2020-02-21 at 11 39 09 PM

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,6 +23,15 @@ and we rely on it to test and ship our apps.
...
 
+## Why do we do open source?
+   
+- To get more feedback. We need outside perspective
+- To make the code easier to change through reusing
+- To share knowledge and solutions and make Android development better
+- To understand better through explanation
+- To supplement our presentations and articles by real production code
+- To make it easier to provide reproducible samples for bugs in external libraries
+
```
Comment (eugene-krivobokov) [inline README.md:34]: As discussed with @sboishtyan, it's not documentation about the product.

File: docs/config.toml [hunk 2]
```diff
@@ -12,10 +12,6 @@ theme = "book"
 # WARNING: publishDir outside the project breaks i18n
 publishDir = "public"
 
-# Disable needless kinds (https://gohugo.io/templates/section-templates/#page-kinds):
-# - We use explicit menu instead of section tree
-disableKinds = ["section"]
```
Comment (eugene-krivobokov) [inline docs/config.toml:17]: Premature optimization

The section is an interesting concept to map files to paths: https://gohugo.io/content-management/organization/
It explains `_index.md` files in the root of a directory.

File: docs/content/ci/LintWorkerApiWorkaround.md [hunk 3]
```diff
@@ -1,41 +0,0 @@
----
```
Comment (eugene-krivobokov) [inline docs/content/ci/LintWorkerApiWorkaround.md:1]: Moved to posts/

File: docs/content/menu/index.md [hunk 4]
```diff
@@ -3,66 +3,65 @@ headless: true
 ---
```
Comment (eugene-krivobokov) [inline docs/content/menu/index.md:3]: I've changed links but it's for the best. 
Now we have simpler structure:

```
content/
  docs/
  menu/
  posts/
```

File: docs/content/menu/index.md [hunk 5]
```diff
@@ -3,66 +3,65 @@ headless: true
...
-    - [QApps]({{< ref "/cd/QApps.md" >}})
-- [Infrastructure (Github project)]({{< ref "/Infrastructure.md" >}})
-
-- [Внешние библиотеки]({{< ref "/ExternalLibraryChangeProcess.md" >}})
+    - [Релиз]({{< ref "/docs/cd/Release.md" >}})
+    - [Артефакты сборки]({{< ref "/docs/cd/Artifacts.md" >}})
+    - [QApps]({{< ref "/docs/cd/QApps.md" >}})
+- [Infrastructure (Github project)]({{< ref "/docs/Infrastructure.md" >}})
+- [Внешние библиотеки]({{< ref "/docs/ExternalLibraryChangeProcess.md" >}})
+- [Blog]({{< relref "/posts" >}})
```
Comment (eugene-krivobokov) [inline docs/content/menu/index.md:67]: Screen Shot 2020-02-21 at 11 39 09 PM

File: docs/content/posts/lint-worker-api-workaround.md [hunk 6]
```diff
@@ -0,0 +1,63 @@
...
+{{<mermaid>}}
+gantt
+	title Lint runs in parallel with instrumentation
+	dateFormat      YYYY-MM-DD
+	axisFormat      %j
+	section         project
+	package         :package, 1000-01-01, 3d
+	instrumentation :after package, 11d
+	lint            :lint, 1000-01-05, 10d
+{{</mermaid>}}
```
Comment (eugene-krivobokov) [inline docs/content/posts/lint-worker-api-workaround.md:58]: Screen Shot 2020-02-21 at 11 53 42 PM

File: docs/content/posts/lint-worker-api-workaround.md [hunk 7]
```diff
@@ -0,0 +1,63 @@
+---
```
Comment (eugene-krivobokov) [inline docs/content/posts/lint-worker-api-workaround.md:1]: This page will be available at https://avito-tech.github.io/avito-android/posts/lint-worker-api-workaround

File: docs/content/posts/lint-worker-api-workaround.md [hunk 8]
```diff
@@ -0,0 +1,63 @@
...
+date: 2020-02-20
+title: Android lint and Gradle worker API workaround
+tags: [gradle, agp, lint]
```
Comment (eugene-krivobokov) [inline docs/content/posts/lint-worker-api-workaround.md:4]: Finally, it makes sense to use tags and categories.

File: README.md (hunk 1 above)
Comment (dsvoronin) [reply README.md:34]: it is clashed with Disclaimer in theme

**Review by dsvoronin — APPROVED** [review]

**Pull request merged**